src/version/getNextVersion.ts

```diff
@@ -4,6 +4,19 @@ const { inc } = semver;
 
 const VERSION: Array<ReleaseType> = ["major", "minor", "patch"];
 
-export default function (version: SemVer, bumpInfo: { level: 0 | 1 | 2 }) {
-  return inc(version.version, VERSION[bumpInfo.level]);
+type BumpInfoWithLevel = { level: 0 | 1 | 2 };
+type BumpInfoWithType = { type: ReleaseType; preid?: string };
+type BumpInfo = BumpInfoWithLevel | BumpInfoWithType;
+
+function getNextVersion(version: SemVer, bumpInfo: BumpInfoWithLevel): string;
+function getNextVersion(version: SemVer, bumpInfo: BumpInfoWithType): string;
+
+function getNextVersion(version: SemVer, bumpInfo: BumpInfo) {
+  if ("level" in bumpInfo) {
+    return inc(version.version, VERSION[bumpInfo.level]);
```

Comment:
Rename VERSION to VERSION_LEVEL to be clear that it's only relevant for the usage with `BumpInfoWithLevel`